src/niscope/metadata/functions_addon.py

```diff
@@ -384,6 +387,125 @@
 digitizer acquires for the specified channel. This function returns
 scaled voltage waveforms.
 
+This function may return multiple waveforms depending on the number of
+channels, the acquisition type, and the number of records you specify.''',
+            'note': 'Some functionality, such as time stamping, is not supported in all digitizers.',
+        },
+    },
+    'FancyRead': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'The instrument handle you obtain from niScope_init that identifies a particular instrument session.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'channelList',
+                'type': 'ViChar[]',
+                'documentation': {
+                    'description': 'The channel to configure.',
+                },
+            },
+            {
+                'direction': 'in',
+                'default_value': None,
+                'name': 'numSamples',
+                'type': 'ViInt32',
+                'documentation': {
+                    'description': 'The maximum number of samples to fetch for each waveform. If the acquisition finishes with fewer points than requested, some devices return partial data if the acquisition finished, was aborted, or a timeout of 0 was used. If it fails to complete within the timeout period, the function throws an exception.',
```

Comment:
throws an exception -> raises

since we are writing Python-specific documentation.

Reply:
Fixed